### PR TITLE
Fixes broken links

### LINKED ISSUE #69: Broken links
Some pages link to broken links.

Running `wget --spider -o wget.log -e robots=off --wait 1 -r -p --debug  'https://www.fastify.io/'`, the following links are broken:
```
Trovati 105 collegamenti rotti.
https://www.fastify.io/docs/v0.35.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v0.42.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.3.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.2.x/Factory
https://www.fastify.io/docs/latest/Middlewares/Request.md
https://www.fastify.io/docs/v0.40.x/Middlewares/Reply.md
https://www.fastify.io/docs/v1.3.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.5.x/Factory
https://www.fastify.io/docs/v0.39.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.6.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.37.x/Middlewares/Reply.md
https://www.fastify.io/docs/v0.41.x/Factory.md
https://www.fastify.io/docs/v1.0.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v0.41.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.0.x/Middlewares/Plugins.md
https://www.fastify.io/docs/master/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v1.4.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v0.36.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.3.x/Middlewares/Reply.md
https://www.fastify.io/docs/v0.42.x/Factory.md
https://www.fastify.io/docs/v0.42.x/Middlewares/Request.md
https://www.fastify.io/docs/v1.3.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.39.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.41.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v0.38.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.1.x/Factory
https://www.fastify.io/docs/v1.5.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v0.43.x/Factory.md
https://www.fastify.io/docs/v0.41.x/Middlewares/Reply.md
https://www.fastify.io/docs/v0.38.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v1.4.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.0.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v0.38.x/Middlewares/Reply.md
https://www.fastify.io/docs/v1.0.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.36.x/Middlewares/Request.md
https://www.fastify.io/manifest.json
https://www.fastify.io/docs/master/Middlewares/Hooks.md
https://www.fastify.io/docs/v0.41.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v0.42.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.3.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v1.2.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v0.38.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.4.x/Factory
https://www.fastify.io/docs/v1.4.x/Middlewares/Reply.md
https://www.fastify.io/docs/v1.0.x/Middlewares/Reply.md
https://www.fastify.io/docs/v0.39.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.5.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.40.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v0.35.x/Middlewares/Hooks.md
https://www.fastify.io/docs/master/Middlewares/Reply.md
https://www.fastify.io/docs/v1.0.x/Factory
https://www.fastify.io/docs/v0.42.x/Middlewares/Reply.md
https://www.fastify.io/docs/v0.37.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v0.35.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.5.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.1.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v0.41.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.39.x/Middlewares/Reply.md
https://www.fastify.io/docs/v1.2.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.38.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.35.x/Middlewares/Reply.md
https://www.fastify.io/docs/latest/Middlewares/Hooks.md
https://www.fastify.io/docs/v0.43.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.2.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v1.4.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.3.x/Factory
https://www.fastify.io/docs/v0.43.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.6.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v1.5.x/Middlewares/Reply.md
https://www.fastify.io/docs/v1.1.x/Middlewares/Reply.md
https://www.fastify.io/docs/v0.36.x/Middlewares/Hooks.md
https://www.fastify.io/docs/latest/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v0.35.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.43.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/latest/Middlewares/Reply.md
https://www.fastify.io/docs/master/Factory.md
https://www.fastify.io/docs/v0.40.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v0.43.x/Middlewares/Reply.md
https://www.fastify.io/docs/v1.1.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v0.37.x/Middlewares/Plugins.md
https://www.fastify.io/docs/master/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.6.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.2.x/Middlewares/Hooks.md
https://www.fastify.io/docs/latest/Factory
https://www.fastify.io/docs/v1.6.x/Factory
https://www.fastify.io/docs/v1.4.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.43.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.36.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/master/Server-Methods.md
https://www.fastify.io/docs/v0.36.x/Middlewares/Reply.md
https://www.fastify.io/docs/latest/Middlewares/Plugins.md
https://www.fastify.io/docs/v1.1.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v1.6.x/Middlewares/Plugins.md
https://www.fastify.io/docs/v0.40.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v0.39.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v1.5.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v1.6.x/Middlewares/Reply.md
https://www.fastify.io/docs/v1.2.x/Middlewares/Reply.md
https://www.fastify.io/docs/v0.40.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.37.x/Middlewares/Hooks.md
https://www.fastify.io/docs/v1.1.x/Middlewares/Request.md
https://www.fastify.io/docs/v0.37.x/Middlewares/Request.md
https://www.fastify.io/docs/master/Middlewares/Request.md
https://www.fastify.io/docs/v0.42.x/Middlewares/'./Reply.md'
https://www.fastify.io/docs/v0.40.x/Factory.md

```

The full logs (sorry it is in italian):
[wget.log](https://github.com/fastify/website/files/2172833/wget.log)

### PR DESCRIPTION
**Pull request by lmammino**
Fixes #69

**WORK IN PROGRESS: Currently not completed**

Current status:

- reduced broken links from `105` to `70`
- Added missing `manifest.json`

Links still broken:

```plain
Found 70 broken links.

http://127.0.0.1:8080/docs/v0.41.x/Factory.md
http://127.0.0.1:8080/docs/v0.40.x/Middlewares/'/docs/v0.40.x/Hooks'
http://127.0.0.1:8080/docs/v1.6.x/Middlewares/'/docs/v1.6.x/Request'
http://127.0.0.1:8080/docs/v0.39.x/Middlewares/'/docs/v0.39.x/Reply'
http://127.0.0.1:8080/docs/v1.4.x/Factory
http://127.0.0.1:8080/docs/latest/Middlewares/'/docs/latest/Hooks'
http://127.0.0.1:8080/docs/v1.2.x/Middlewares/'/docs/v1.2.x/Hooks'
http://127.0.0.1:8080/docs/v0.43.x/Factory.md
http://127.0.0.1:8080/docs/v1.0.x/Middlewares/'/docs/v1.0.x/Reply'
http://127.0.0.1:8080/docs/v1.1.x/Middlewares/'/docs/v1.1.x/Reply'
http://127.0.0.1:8080/docs/v0.39.x/Middlewares/'/docs/v0.39.x/Hooks'
http://127.0.0.1:8080/docs/v0.37.x/Middlewares/'/docs/v0.37.x/Reply'
http://127.0.0.1:8080/docs/v1.0.x/Middlewares/'/docs/v1.0.x/Request'
http://127.0.0.1:8080/docs/v1.0.x/Middlewares/'/docs/v1.0.x/Hooks'
http://127.0.0.1:8080/docs/v1.1.x/Middlewares/'/docs/v1.1.x/Hooks'
http://127.0.0.1:8080/docs/master/Factory.md
http://127.0.0.1:8080/docs/v0.38.x/Middlewares/'/docs/v0.38.x/Request'
http://127.0.0.1:8080/docs/v0.43.x/Middlewares/'/docs/v0.43.x/Reply'
http://127.0.0.1:8080/docs/v0.35.x/Middlewares/'/docs/v0.35.x/Request'
http://127.0.0.1:8080/docs/v0.37.x/Middlewares/'/docs/v0.37.x/Hooks'
http://127.0.0.1:8080/docs/v1.0.x/Factory
http://127.0.0.1:8080/docs/v0.35.x/Middlewares/'/docs/v0.35.x/Reply'
http://127.0.0.1:8080/docs/v1.3.x/Factory
http://127.0.0.1:8080/docs/v0.42.x/Middlewares/'/docs/v0.42.x/Request'
http://127.0.0.1:8080/docs/v0.43.x/Middlewares/'/docs/v0.43.x/Hooks'
http://127.0.0.1:8080/docs/v0.41.x/Middlewares/'/docs/v0.41.x/Reply'
http://127.0.0.1:8080/docs/v0.35.x/Middlewares/'/docs/v0.35.x/Hooks'
http://127.0.0.1:8080/docs/v1.1.x/Middlewares/'/docs/v1.1.x/Request'
http://127.0.0.1:8080/docs/latest/Factory
http://127.0.0.1:8080/docs/v1.6.x/Factory
http://127.0.0.1:8080/docs/v0.41.x/Middlewares/'/docs/v0.41.x/Hooks'
http://127.0.0.1:8080/docs/v1.2.x/Middlewares/'/docs/v1.2.x/Request'
http://127.0.0.1:8080/docs/v0.37.x/Middlewares/'/docs/v0.37.x/Request'
http://127.0.0.1:8080/docs/v0.38.x/Middlewares/'/docs/v0.38.x/Reply'
http://127.0.0.1:8080/docs/master/Logging/Factory.md
http://127.0.0.1:8080/docs/v0.41.x/Middlewares/'/docs/v0.41.x/Request'
http://127.0.0.1:8080/docs/master/Middlewares/'/docs/master/Reply'
http://127.0.0.1:8080/docs/v1.2.x/Factory
http://127.0.0.1:8080/docs/v1.3.x/Middlewares/'/docs/v1.3.x/Request'
http://127.0.0.1:8080/docs/latest/Middlewares/'/docs/latest/Request'
http://127.0.0.1:8080/docs/v0.38.x/Middlewares/'/docs/v0.38.x/Hooks'
http://127.0.0.1:8080/docs/v0.36.x/Middlewares/'/docs/v0.36.x/Reply'
http://127.0.0.1:8080/docs/v1.5.x/Middlewares/'/docs/v1.5.x/Reply'
http://127.0.0.1:8080/docs/v1.6.x/Middlewares/'/docs/v1.6.x/Reply'
http://127.0.0.1:8080/docs/master/Middlewares/'/docs/master/Hooks'
http://127.0.0.1:8080/docs/v1.4.x/Middlewares/'/docs/v1.4.x/Request'
http://127.0.0.1:8080/docs/v0.42.x/Middlewares/'/docs/v0.42.x/Reply'
http://127.0.0.1:8080/docs/v0.36.x/Middlewares/'/docs/v0.36.x/Hooks'
http://127.0.0.1:8080/docs/v1.5.x/Factory
http://127.0.0.1:8080/docs/v1.5.x/Middlewares/'/docs/v1.5.x/Hooks'
http://127.0.0.1:8080/docs/master/Server-Methods.md
http://127.0.0.1:8080/docs/v1.6.x/Middlewares/'/docs/v1.6.x/Hooks'
http://127.0.0.1:8080/docs/v1.3.x/Middlewares/'/docs/v1.3.x/Reply'
http://127.0.0.1:8080/docs/v1.4.x/Middlewares/'/docs/v1.4.x/Reply'
http://127.0.0.1:8080/docs/v0.43.x/Middlewares/'/docs/v0.43.x/Request'
http://127.0.0.1:8080/docs/v1.1.x/Factory
http://127.0.0.1:8080/docs/v0.40.x/Middlewares/'/docs/v0.40.x/Request'
http://127.0.0.1:8080/docs/v0.42.x/Middlewares/'/docs/v0.42.x/Hooks'
http://127.0.0.1:8080/docs/v0.40.x/Middlewares/'/docs/v0.40.x/Reply'
http://127.0.0.1:8080/docs/v0.40.x/Factory.md
http://127.0.0.1:8080/docs/v0.39.x/Middlewares/'/docs/v0.39.x/Request'
http://127.0.0.1:8080/docs/v1.3.x/Middlewares/'/docs/v1.3.x/Hooks'
http://127.0.0.1:8080/docs/v1.4.x/Middlewares/'/docs/v1.4.x/Hooks'
http://127.0.0.1:8080/docs/latest/Middlewares/'/docs/latest/Reply'
http://127.0.0.1:8080/docs/v1.2.x/Middlewares/'/docs/v1.2.x/Reply'
http://127.0.0.1:8080/docs/v1.5.x/Middlewares/'/docs/v1.5.x/Request'
http://127.0.0.1:8080/docs/master/Factory/Logging.md
http://127.0.0.1:8080/docs/v0.42.x/Factory.md
http://127.0.0.1:8080/docs/v0.36.x/Middlewares/'/docs/v0.36.x/Request'
http://127.0.0.1:8080/docs/master/Middlewares/'/docs/master/Request'
```

### CODE REVIEW

File: src/scripts/processReleases.js [hunk 1]
```diff
@@ -121,21 +119,22 @@ ${content}`
...
+    Links remapping rules:
+    /https:\/\/github.com\/fastify\/fastify\/blob\/master\/docs/ -> /docs/[VERSION]
+    [XXXX](/docs/VVVV/Ecosystem.md) -> [XXXX](/ecosystem)
+    [XXXX](/docs/VVVV/YYYY.md) -> [XXXX](/docs/VVVV/YYYY)
+    [XXXX]('./YYYY' "ZZZZ") -> [XXXX]('/docs/[VERSION]/YYYY' "ZZZZ")
+    [XXXX](./YYYY "ZZZZ") -> [XXXX]('/docs/[VERSION]/YYYY' "ZZZZ")
   */
   const ecosystemLinkRx = /\(\/docs\/[\w\d.-]+\/Ecosystem\.md\)/gi
   const docInternalLinkRx = /\(\/docs\/[\w\d.-]+\/[\w\d-]+(.md)/gi
+  const relativeLinksWithLabel = /\('?(\.\/)([\w\d.-]+)(.md)'?\s"([\w\d.-]+)"\)/gi
```
Comment (allevo) [inline src/scripts/processReleases.js]: `\s` => `\s+`?

**Review by mcollina — COMMENTED** [review]
Can you check it Factory.md is now included?

**Comment by lmammino** [comment]
@mcollina, it is included in current master but not it all the available releases. I am trying to figure out if I can somehow "force" it into those.

**Comment by mcollina** [comment]
@lmammino I can't see that in the v1.7.0 release. Let's see if it appears in the upcoming v1.8.0 release.

**Comment by mcollina** [comment]
What's missing here, can we land this?

**Comment by lmammino** [comment]
@matteocollina we could start to merge this. It's not gonna solve all the broken links but reduce it, which is still delivering some value.

**Comment by lmammino** [comment]
Just for clarity, all the links on the description are still broken.

**Comment by lmammino** [comment]
Creating a new issue to be sure we get rid of all the other broken links!

**Pull request merged**